exporter/observiqexporter/client.go

```diff
@@ -201,6 +202,10 @@ func buildClient(config *Config, logger *zap.Logger, buildInfo component.BuildIn
 			Transport: &http.Transport{
 				Proxy:           http.ProxyFromEnvironment,
 				TLSClientConfig: tlsCfg,
+				Dial: (&net.Dialer{
+					Timeout: 10 * time.Second,
+				}).Dial,
+				TLSHandshakeTimeout: 10 * time.Second,
```

Comment:
Alright, I know this is what we're doing in bpagent, but, should this be at a flat 10 seconds, or based off of configuration? I'm good to merge & use this as is, but I'm wondering what the rationale is -- the default timeout is no timeout, so it should just timeout with the http timeout.

Also, looks like the dial option is deprecated: https://pkg.go.dev/net/http#Transport